app/domain/mail_relay/lists.rb

```diff
@@ -124,10 +124,18 @@ def prepare_bounced_message(list_address, sender_address)
       message.smtp_envelope_from = env_sender
     end
 
+    def domain_is_valid?(domain)
```

Comment:
Please try to avoid auxiliary verbs such as `is` in predicate methods.

```suggestion
    def valid_domain?(domain)
```